src/java/org/apache/cassandra/service/accord/AccordKeyspace.java

```diff
@@ -231,6 +276,7 @@ private static ColumnMetadata getColumn(TableMetadata metadata, String name)
               "accord commands per key",
               "CREATE TABLE %s ("
               + "store_id int, "
+              + "key_hash blob, " // can't use "token" as this is restricted word in CQL
```

Comment:
Nit, since we generally call them `Token` it might be clearer to call it `key_token`?

Reply:
I used `token` before, which is a reserved word... I could def use `key_token`